doc/userguide/output/eve/eve-json-format.rst

```diff
@@ -1056,6 +1056,8 @@ In addition to this, custom logging also allows the following fields:
 * "client_handshake": structure containing "version", "ciphers" ([u16]), "exts" ([u16]), "sig_algs" ([u16]),
   for client hello supported cipher suites, extensions, and signature algorithms,
   respectively, in the order that they're mentioned (ie. unsorted)
+* "server_handshake": structure containing "version", "chosen cipher", "exts" ([u16]), for server hello
+  in the order that they're mentioned (ie. unsorted)
```

Comment:
Should the info about these new fields be also present in the yaml section about TLS custom logging? (https://github.com/OISF/suricata/blob/c13f85f18d859603c1a9f55f3310310f9c8d19c9/suricata.yaml.in#L291)

Reply:
Yes, yes they should! I will include these also in the next revision, thanks :)